tests/test_utilities.py

```diff
@@ -923,6 +910,169 @@ def test_exp_val_hamiltonian_termwise(self):
                 corr_matrix[j], num_corr_matrix[j]
             ), f"Computed correlation matrix is incorrect"
 
+    def test_calculate_calibration_factors(self):
+        """
+        Tests the function that computes the calibration factors.
+
+        The test consists in ...
```

Comment:
[tag] To be completed

Reply:
Completed. However, should I mode the comments from the code to the doc strings?